### PR TITLE
Add asterisk support , flowing gherkin specification.

### PR DESCRIPTION
**Pull request by lxbzmy**

### CODE REVIEW

File: src/Behat/Gherkin/Lexer.php [hunk 1]
```diff
@@ -325,7 +325,7 @@ protected function getKeywords($type)
                 foreach (explode('|', $keywords) as $keyword) {
                     $padded[] = false !== mb_strpos($keyword, '<', 0, 'utf8')
                         ? mb_substr($keyword, 0, -1, 'utf8') . '\s*'
-                        : $keyword . '\s+';
+                        : preg_quote($keyword) . '\s+';
```
Comment (stof) [inline src/Behat/Gherkin/Lexer.php]: please don't duplicate the work of the existing PR, especially given that your fix is incomplete

File: i18n.php [hunk 2]
```diff
@@ -921,7 +921,7 @@
     'but' => 'Nhưng',
     'examples' => 'Dữ liệu',
     'feature' => 'Tính năng',
-    'given' => 'Biết|Cho',
+    'given' => '*|Biết|Cho',
```
Comment (stof) [inline i18n.php]: this file must not be edited by hand. We have a script generating it (which needs to be updated btw, which is on my TODO list). Manual updates on this file will be rejected

File: src/Behat/Gherkin/Lexer.php (hunk 1 above)
Comment (lxbzmy) [inline src/Behat/Gherkin/Lexer.php]: Test case passed.

File: i18n.php (hunk 2 above)
Comment (lxbzmy) [inline i18n.php]: update_i18n 
https://raw.github.com/cucumber/gherkin/master/lib/gherkin/i18n.json 
404 not found.

Comment (lxbzmy) [inline i18n.php]: change to this?https://github.com/cucumber/gherkin/blob/master/ruby/lib/gherkin/gherkin-languages.json

File: src/Behat/Gherkin/Lexer.php (hunk 1 above)
Comment (stof) [inline src/Behat/Gherkin/Lexer.php]: yeah, for the asterisk. But it misses some escaping

File: i18n.php (hunk 2 above)
Comment (stof) [inline i18n.php]: See the discussion in https://github.com/Behat/Gherkin/issues/103

I will update the script this weekend to read the new i18n file.

**Comment by everzet** [comment]
Can you rebase, please?

**Comment by stof** [comment]
Closing because #112 actually made it (and with full support)

**Pull request closed without merging**